be/src/vec/data_types/serde/data_type_jsonb_serde.cpp

```diff
@@ -83,7 +83,10 @@ Status DataTypeJsonbSerDe::serialize_one_cell_to_json(const IColumn& column, int
 
     const StringRef& s = assert_cast<const ColumnString&>(*ptr).get_data_at(row_num);
     if (s.size > 0) {
-        bw.write(s.data, s.size);
+        std::string str = JsonbToJson::jsonb_to_json_string(s.data, s.size);
+        bw.write(str.c_str(), str.size());
+    } else {
+        bw.write("\\N", 2);
```

Comment:
You can use `NULL_IN_CSV_FOR_ORDINARY_TYPE`, rather than `\\N`

Reply:
done